buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -1,23 +1,25 @@
 package com.osacky.flank.gradle
 
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.MapProperty
+import org.gradle.api.provider.Property
 import org.gradle.internal.impldep.com.google.common.annotations.VisibleForTesting
 
 internal class YamlWriter {
 
   internal fun createConfigProps(config: FladleConfig, base: FlankGradleExtension): String {
-    if (base.projectId == null) {
+    if (base.projectId.orNull == null) {
       check(base.serviceAccountCredentials.isPresent) { "ServiceAccountCredentials in fladle extension not set. https://github.com/runningcode/fladle#serviceaccountcredentials" }
     }
     check(base.debugApk.isPresent) { "debugApk must be specified" }
-    check(base.instrumentationApk.isPresent xor !base.roboScript.isNullOrBlank()) {
+    check(base.instrumentationApk.isPresent xor base.roboScript.isPresent) {
```

Comment:
the previous logic checked if the `roboScript` was not null and not blank. Should we check that here as well?

Reply:
TBH I based my implementation on `instrumentationApk` check, but yeah, for now we should check for path being blank as well.

FYI, this will change with https://github.com/runningcode/fladle/issues/165